src/main/webapp/static/js/controllers/scheduling_controller.js

```diff
@@ -46,7 +46,7 @@ function SchedulingController(SchedulingService, $log){
 	}
 	
 	function submit() {
-		if(vm.job.id === null) {
+		if(vm.job.id === undefined) {
```

Comment:
javascript objects are "truthy" you can just use` if (!vm.job.id)` in this case, and if it is null or undefiend it will result in "true".